contract/src/error.rs

```diff
@@ -142,6 +142,9 @@ pub enum NFTCoreError {
     InvalidAccessKeyName = 137,
     InvalidCheckForUpgrade = 138,
     InvalidNamedKeyConvention = 139,
+    InvalidTokenEvent = 140,
+    MissingEventsMode = 141,
+    InvalidEventsMode = 142,
```

Comment:
```suggestion
    MissingEventsMode = 140,
    InvalidEventsMode = 141,
```
As per previous comment, this chnage probably triggers some tests